### ISSUE TITLE
Rethinking warped animal carrier

### ISSUE THREAD

**Issue by Coach-mdl** [TGWeaver/CDDA-Sky-Islands#67]
I had my first use of it today and I was not expecting the full weight of the cow to bear down on me. It wasn't instant, there's a bug that doesn't properly register the items weight until you add/remove something from your inventory, so I was able to get the cow home with some unusual methods.

Assuming it's intended to be used to take back any tamed animal, I don't feel like this is the best solution for this as it requires inhuman strength or a vehicle that has enough space in the trunk (God forbid you catch a large dinosaur).

Removing the weight doesn't exactly solve the issue either. Since their fairly light when empty you could theoretically take a whole farm home with you without weight being a factor.

I suggest making it a single use item, removing (or lowering) the weight when full and allowing only one to be in your inventory at a time.

Hopefully this helps.

**Comment by adamkad1** [comment]
Yeah, with hauler harness only halving weights now, this is rather problematic
